how-to/how-to-rename-field-in-content-type.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to rename a field in content type
+description: Rename a field in content type using mongodb '$rename' operator
+type: how-to
+page_title: Rename a field in a content type
+slug: rename-field-in-content-type
+position:
+tags: platform-core, rename-field, backendservices, content-type
+teampulseid: 
+ticketid:  1046311
+pitsid:
+aws: Amazon Web Services

```

Comment:
Used a previous article as a template, was left over from that article. The article is [here](https://github.com/telerik/Platform-Knowledge-Base/blob/master/how-to/how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md).